packages/client/src/links/httpFormDataLink.ts

```diff
@@ -1,29 +0,0 @@
-import { httpLinkFactory } from './httpLink';
-import type { Requester } from './internals/httpUtils';
-import { httpRequest } from './internals/httpUtils';
-
-const formDataRequester: Requester = (opts) => {
-  if (opts.type !== 'mutation') {
-    // TODO(?) handle formdata queries
-    throw new Error('We only handle mutations with formdata');
-  }
-  return httpRequest({
-    ...opts,
-    getUrl() {
-      return `${opts.url}/${opts.path}`;
-    },
-    getBody(opts) {
-      if (!('input' in opts)) {
-        return undefined;
-      }
-      if (!(opts.input instanceof FormData)) {
-        throw new Error('Input is not FormData');
-      }
-      return opts.input;
-    },
-  });
-};
-
-export const experimental_formDataLink = httpLinkFactory({
-  requester: formDataRequester,
-});
```

Comment:
idk if we do migration guides for experimental things but might as well mention that this has been removed?

Reply:
Already done:

<img width="1015" alt="image" src="https://github.com/trpc/trpc/assets/8896153/8ef2689f-c121-4236-84b8-410cb6a9200c">